coderd/notifications/types/labels.go

```diff
@@ -0,0 +1,72 @@
+package types
+
+import (
+	"fmt"
+)
+
+// Labels represents the metadata defined in a notification message, which will be used to augment the notification
+// display and delivery.
+type Labels map[string]string
```

Comment:
Should we add extra unit tests for basic operations on Labels?

Reply:
Yeah, good idea.